examples/test_scr_poststage

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# This script generates checkpoints on the IBM burst buffer disk, and then
+# starts transferring them to GPFS before exiting.  Then, the scr_poststage
+# script gets run after the transfers finish, and finalizes them.
+#
+# Example:
+#
+# # Allocate one node with 100GB burst buffer, run test_scr_poststag to
+# # create and begin transferring checkpoints, and after the transfers finish
+# # (after the job ends), call scr_poststage to finalize them.
+# bsub -nnodes 1 -stage storage=100:out=,scr_poststage lexec jsrun -r 1 test_scr_poststage
+#
+
+# Where do you want to write your checkpoints to?  This should be somewhere in
+# on the burst buffer filesystem.
+
+BB_DIR="$BBPATH/my_checkpoints"
+
```

Comment:
Let's simplify the test to use ``$BBPATH`` rather than some directory in ``$BBPATH`` since that's how we'll be recommending that users configure things.
```suggestion
```